src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinFeature.kt

```diff
@@ -0,0 +1,19 @@
+package com.fasterxml.jackson.module.kotlin
+
+import com.fasterxml.jackson.core.util.JacksonFeature
+import java.util.*
+
+enum class KotlinFeature(private val enabledByDefault: Boolean, private val mask: Int) : JacksonFeature {
+
+    NullToEmptyCollection(enabledByDefault = false, mask = 1),
+    NullToEmptyMap(enabledByDefault = false, mask = 2),
+    NullIsSameAsDefault(enabledByDefault = false, mask = 4),
+    SingletonSupport(enabledByDefault = false, mask = 8),
+    StrictNullChecks(enabledByDefault = false, mask = 16);
+
+    val bitSet: BitSet = mask.toBitSet()
+
+    override fun enabledByDefault() = enabledByDefault
+    override fun getMask() = mask
+    override fun enabledIn(flags: Int) = bitSet.intersects(flags.toBitSet())
```

Comment:
I may be wrong but this would seem like a relatively expensive operation, constructing bitset for every access?

Reply:
I wonder if there's even a good reason for `KotlinFeature` to implement `JacksonFeature`? This function is never even called in this module.

Reply:
No need to implement it, and probably not much benefit -- it (may) make(s) some sense for features implemented by core components but less so for module-specific ones. The idea being that if core components need to expose features, it can be essentially tag interface.

So feel free to ignore it.

Reply:
I dropped implementing `JacksonFeature`.